restapi/stories.go

```diff
@@ -15,6 +15,16 @@ import (
 func (api api) ReturnAllStories(w http.ResponseWriter, r *http.Request) render.Renderer {
 	var stories []models.Story
 
+	for i := 0; i < len(stories); i++ {
```

Comment:
should this be in `CreateStories` endpoint instead? 

if i'm not wrong, this code is setting the `longitude` and `latitude` fields of the story based on its `CurrentCity` field - so i think we should be doing this when creating stories, not when returning them

Reply:
Yes you are right. It should be in createStories. I will fix it.